_includes/header.html

```diff
@@ -21,6 +21,7 @@
           <div class="trigger">
               <a class="header-link page-link" href="/get-started/">Get Started</a>
               <a class="page-link" href="/guides/">Guides</a>
+              <a class="page-link" href="/blog/">Blog</a>
```

Comment:
Maybe `Blogs` to be consistent with `Guides`?

Reply:
Kind of a philosophical question... each of the items within that collection is a `Post` in my mind, not a singular `Blog`. From your POV each one is a `Blog` and so the collection header is `Blogs` :-)

I'd be fine with either way personally but I'm inclined to leave it as `Blog` since that's how James phrased it originally, which I believe he's also responsible for doing on the Quarkus website.

Reply:
ohhh right! Interesting tho. I think we can keep the original :D